behave_django/__init__.py

```diff
@@ -5,4 +5,4 @@
 __email__ = 'mixxorz@gmail.com'
 __url__ = 'https://github.com/behave/behave-django'
 __license__ = 'MIT'
-__version__ = '1.3.0'
+__version__ = '1.3.1'
```

Comment:
That's more than just a bug fix or something. Let's make it `1.4.0`, what do you think?